myhpi/settings.py

```diff
@@ -218,6 +218,14 @@
     constants.ERROR: "alert-danger",
 }
 
+# Create a localsettings.py to override settings per machine or user, e.g. for
```

Comment:
we are using django-environ in this project to circumvent the hassle with a localsettings file